core/src/main/java/com/radixdlt/consensus/epoch/EpochsConsensusModule.java

```diff
@@ -96,19 +93,39 @@
 public class EpochsConsensusModule extends AbstractModule {
   @Override
   protected void configure() {
+    bind(PacemakerState.class).in(Scopes.SINGLETON);
+    bind(PacemakerReducer.class).to(PacemakerState.class);
+    bind(ExponentialPacemakerTimeoutCalculator.class).in(Scopes.SINGLETON);
+    bind(PacemakerTimeoutCalculator.class).to(ExponentialPacemakerTimeoutCalculator.class);
+
     OptionalBinder.newOptionalBinder(
         binder(), EpochManager.class); // So that this is consistent with tests
     bind(EpochManager.class).in(Scopes.SINGLETON);
     var eventBinder =
         Multibinder.newSetBinder(binder(), new TypeLiteral<Class<?>>() {}, LocalEvents.class)
             .permitDuplicates();
+
+    // Non-epoched local events
+    eventBinder.addBinding().toInstance(BFTRebuildUpdate.class);
+    eventBinder.addBinding().toInstance(BFTInsertUpdate.class);
+    eventBinder.addBinding().toInstance(Proposal.class);
+    eventBinder.addBinding().toInstance(Vote.class);
+    eventBinder.addBinding().toInstance(LedgerUpdate.class);
+    eventBinder.addBinding().toInstance(VertexRequestTimeout.class);
+
+    // Epoched local events
     eventBinder.addBinding().toInstance(EpochRoundUpdate.class);
     eventBinder.addBinding().toInstance(EpochProposalRejected.class);
-    eventBinder.addBinding().toInstance(VertexRequestTimeout.class);
-    eventBinder.addBinding().toInstance(LedgerUpdate.class);
+    eventBinder.addBinding().toInstance(EpochLocalTimeoutOccurrence.class);
     eventBinder.addBinding().toInstance(Epoched.class);
   }
 
+  @Provides
+  @Singleton
```

Comment:
It feels really wrong that we need this in an EpochsConsensusModule (it feels like we should have one per epoch instead?) - could this be removed / fixed?

Reply:
ohh I've missed it (and the PacemakerState bound in configure()), fixed. I believe we now only need the (initial) BFTConfiguration - which I'd also like to clean up at some point :)